src/dash/Dashboard.jsx

```diff
@@ -8,14 +8,13 @@ import { Provider } from 'react-redux';
 import store from '../redux/store.js';
 import AboutUs from '../home/AboutUs.jsx';
 import {Router, Route, browserHistory} from 'react-router';
-import { updateUser } from '../redux/actions.js';
+import { updateUser } from '../redux/actions/user_actions.js';
 import { connect } from 'react-redux';
 import axios from 'axios';
 
 export class DashComp extends React.Component {
   constructor(props) {
     super(props);
-
   }
 
```

Comment:
Also in this component you have mapStateToProps returning the whole state but you're never using it. Instead you could have it just return an empty object.